packages/grpc-js/src/deadline-filter.ts

```diff
@@ -62,7 +62,13 @@ export class DeadlineFilter extends BaseFilter implements Filter {
                        'connectivityStateChanged', handleStateChange);
                  }
                };
+               const handleStatus = () => {
```

Comment:
Should `handleStatus()` and `handleStateChange()` remove the new `'status'` listener?

Reply:
You're right. I added that in `handleStateChange`, but it shouldn't be needed in `handleStatus` because that is a `once` listener.